crypto/evp/p_lib.c

```diff
@@ -667,6 +667,25 @@ int EVP_PKEY_get_default_digest_nid(EVP_PKEY *pkey, int *pnid)
     return evp_pkey_asn1_ctrl(pkey, ASN1_PKEY_CTRL_DEFAULT_MD_NID, 0, pnid);
 }
 
+int EVP_PKEY_supports_digest_nid(EVP_PKEY *pkey, int nid)
+{
+    int rv, default_nid;
+
+    rv = evp_pkey_asn1_ctrl(pkey, ASN1_PKEY_CTRL_SUPPORTS_MD_NID, nid, NULL);
+    if (rv == -2)
+    {
+        /* If there is a mandatory default digest and this isn't it, then
+         * the answer is 'no'. */
```

Comment:
Style wise, block comments in our code should look like this:

``` C
        /*
         * If there is a mandatory default digest and this isn't it, then
         * the answer is 'no'.
         */
```

Reply:
Done. Thanks.